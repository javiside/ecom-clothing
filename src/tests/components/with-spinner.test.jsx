import React from "react";
import { shallow } from "enzyme";

import WithSpinner from "../../components/with-spinner";

describe("WithSpinner", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<WithSpinner />)).toMatchSnapshot();
  });
});
