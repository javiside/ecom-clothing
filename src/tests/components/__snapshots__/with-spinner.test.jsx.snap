// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithSpinner should mount the component correctly 1`] = `<Component />`;
